DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h

```diff
@@ -33,18 +36,21 @@ struct BCMask {
 struct CTPInput {
   CTPInput() = default;
   std::string name;
-  o2::detectors::DetID::ID detID;
+  std::string level;
+  std::string detName;
   std::uint64_t inputMask;
```

Comment:
is this detectors input mask? If yes, you could probably use DetID::mask_t

Reply:
No, inputMask is position of CTP input in CTP hardware